.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+15
```

Comment:
GHA jobs will now read this file to determine what Node version to install. However, the `lts/*` alias will not work with the Action used to setup Node as aliases are not supported at the moment (see https://github.com/actions/setup-node/issues/26).

As for requiring Node v15, that version comes bundled with npm v7 by default.